payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/BsbSpec.kt

```diff
@@ -0,0 +1,606 @@
+package com.stripe.android.ui.core.elements
+
+import com.stripe.android.view.BecsDebitBanks
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+internal object BsbSpec : SectionFieldSpec(IdentifierSpec.Generic("bsb_number")) {
+    fun transform(): SectionFieldElement =
+        SimpleTextElement(
+            this.identifier,
+            TextFieldController(BsbConfig(banks))
+        )
+}
+
+internal val banks: List<BecsDebitBanks.Bank> = listOf(
```

Comment:
Is this list static? If not it should be loaded from a json like the [BankRepository](https://github.com/stripe/stripe-android/blob/d6c44dabe9fefbcee93d71f0380b37b0848cbbdc/payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/BankRepository.kt), loaded by the `AsyncResourceRepository`. These are currently local but we intend to have a remote resources repository so that the list can be updated without the need to release a new SDK version.
It's fine to do that in a future PR.

Reply:
Yeah, Michelle mentioned that this would need to change down the road. I think the decision was it would be out of scope for our /dev/start project

Reply:
yeah, keeping this out of scope for the /dev/start.  I think I will need to do a refactor soon that reads all the specs from json rather than just the bank and address.  

great point bruno, thanks!